src/MBC_RegistrationMobile_Service_MobileCommons.php

```diff
@@ -30,7 +38,21 @@ public function __construct($message) {
     parent::__construct($message);
     $this->mobileServiceName = 'Mobile Commons';
     $this->mbMobileCommons = $this->mbConfig->getProperty('mbMobileCommons');
+
+    // Cache gambit campaigns.
     $this->gambit = $this->mbConfig->getProperty('gambit');
+    $gambitCampaigns = $this->gambit->getAllCampaigns();
```

Comment:
Does this query `/campaigns` without the `campaignbot` parameter? The GET `campaigns` endpoint doesn't have any pagination, so it will only [return the 1st 20 documents](https://docs.mongodb.com/manual/reference/method/db.collection.find/#cursor-handling) found by default right now.